lib/cacheable/response_cache_handler.rb

```diff
@@ -2,12 +2,10 @@
 
 module Cacheable
   class ResponseCacheHandler
-    attr_accessor :key_data, :version_data, :block, :cache_store
+    attr_accessor :key_data, :version_data, :block, :cache_store, :headers, :env, :cache_age_tolerance_in_seconds,
+      :serve_unversioned, :force_refill_cache
```

Comment:
should these be only `attr_writer`s or do they need be exposed in the public API of the cache handler?